ui/pages/confirmation/confirmation.js

```diff
@@ -484,7 +489,31 @@ export default function ConfirmationPage({
           )
           ///: END:ONLY_INCLUDE_IF
         }
-        <MetaMaskTemplateRenderer sections={templatedValues.content} />
+        {
+          ///: BEGIN:ONLY_INCLUDE_IF(snaps)
+          isSnapDialog ? (
```

Comment:
We can't use this condition as some of the account confirmations are technically snap dialogs.

Reply:
Ah right :(

Reply:
Let's introduce a new condition for whether to use snap UI or not